pure.zsh

```diff
@@ -292,11 +292,12 @@ prompt_pure_async_vcs_info() {
 prompt_pure_async_git_dirty() {
 	setopt localoptions noshwordsplit
 	local untracked_dirty=$1
+	local untracked_git_mode=$(command git config --get status.showUntrackedFiles) || 'normal'
```

Comment:
`|| 'normal'` does not work.

You need to validate the output of `git config` since it could be any string that the user puts in their config. I suggest the following:
```sh
untracked_git_mode="$(command git config --get status.showUntrackedFiles)"
if [[ "$untracked_git_mode" != 'no' ]]; then
    untracked_git_mode='normal'
fi
```

Reply:
that makes sense! I've force pushed a new commit